app/views/users/_form.html.erb

```diff
@@ -73,12 +73,49 @@
     <div class='tab-pane' id='roles'>
       <%= checkbox_f f, :admin if User.current.can_change_admin_flag? %>
       <%= multiple_checkboxes f, :roles, @user, Role.givable.for_current_user,
-                               { :label => _('Roles')}, {:disabled => @editing_self ? Role.givable.for_current_user.pluck(:id) : false } %>
+                              { :label => _('Roles')}, {:disabled => @editing_self ? Role.givable.for_current_user.pluck(:id) : false } %>
+      <% usergroups = @user.cached_usergroups.includes(:roles).uniq %>
+      <% if usergroups.any? %>
+        <div class="form-group">
+          <label class="col-md-2 control-label" for="roles"><%= _("Roles from user groups") %></label>
+          <div class="col-md-5">
+            <div class="dropdown">
+              <button class="btn btn-default dropdown-toggle" type="button" id="dropdownMenu1" data-toggle="dropdown">
```

Comment:
@dhlavac I guess you forgot to change `dropdownMenu1` to `usergroupsDropdownMenuBtn` here.